Documentation/bitbang.md

```diff
@@ -1,7 +1,7 @@
 Bitbang
 =======
 
-![Bitbang session](images/RawSpi.png)
+![Bitbang session](https://raw.githubusercontent.com/BusPirate/Bus_Pirate/master/Documentation/images/Rawspi.png)
```

Comment:
I think hardcoding some GitHub-URL is very fragile.
Wouldn't `images/Rawspi.png` (note the casing) work?

Reply:
Updated the commit to use a relative image link I believe.